ui/js/components/perf/compare.js

```diff
@@ -71,7 +71,8 @@ treeherder.component('phCompareTable', {
 
         ctrl.updateFilteredTestList = function () {
             ctrl.filteredResultList = {};
-            _.forEach(ctrl.compareResults, function (result, key) {
+            Object.entries(ctrl.compareResults).forEach((entry) => {
```

Comment:
This would benefit from destructuring like this:

    Object.entries(ctrl.compareResults).forEach(([key, result]) => {

And then you don't need line 75.  :)  Would you go ahead and make that change?  Then it'll be ready to merge.

Thanks!